client/src/redux/settings/danger-zone-saga.js

```diff
@@ -7,15 +7,26 @@ import { postResetProgress, postDeleteAccount } from '../../utils/ajax';
 import { actionTypes as appTypes } from '../action-types';
 import { deleteAccountError, resetProgressError } from './';
 
+// Dealy funtion
+function wait(ms) {
+  var start = new Date().getTime();
+  var end = start;
+  while (end < start + ms) {
+    end = new Date().getTime();
+  }
+}
+
 function* deleteAccountSaga() {
   try {
     yield call(postDeleteAccount);
     yield put(
       createFlashMessage({
         type: 'info',
-        message: 'flash.account-deleted'
+        message: 'Your account has been successfully deleted'
```

Comment:
This message does not need to be changed, the value is stored in a key. 
```suggestion
        message: 'flash.account-deleted'
```

Reply:
sure thing!